### ISSUE TITLE
Honor Edge set as default browser/ company wide installation

### ISSUE THREAD

**Issue by fashberg** [rcmaehl/MSEdgeRedirect#306]
### Description of the new feature / enhancement

Hey!

the latest versions of MS Outlook is opening all Mail-HTTP-Links in edge, ignoring completely the default browser! 
MS Teams will follow.

This brazen behavior can be set to "Use Default Browser" with Group-Policy or Office-Online-Console, but _only_ with Enterprise Office Subscription, not with the Business Editions (<300 User).

Soon nearly all small to medium business users will start to use Edge.

So i'm looking for a solution to rollout a fix. MSEdgeRedirect installed via GPO with "/wingetinstall" works fine.

But we have some users who really want to use edge. I want to honor their decision.  
Now they get "You must use a different default browser" or opening a PDF fails completely (if edge is default for PDF).

Sounds weird having installed MSEdgeRedirect when you want to really use Edge, but yes, in this situation it makes sense.

Is it possible to get this managed?

Best Regards

Folke

### Scenario when this would be used?

MS Office Business
User has set Edge as Default for web and/or PDF
User really wants to use edge, but the admin has rolled out MSER

**Comment by rcmaehl** [comment]
Hello,

If possible, try the following script on a test PC and let me know if it allows using the default browser in Outlook.

If not, I'll look into getting this added for you

**Comment by fashberg** [comment]
Hey Robert!

Script itself adds that entries to registry, but Outlook switches instantly back to "edge".
As i've stated above, for me this looks like a signed value, so the signature is wrong if shared to an other user/other pc.
Microsoft knows how to protect this.

**Comment by rcmaehl** [comment]
> Hey Robert!
> 
> Script itself adds that entries to registry, but Outlook switches instantly back to "edge". As i've stated above, for me this looks like a signed value, so the signature is wrong if shared to an other user/other pc. Microsoft knows how to protect this.

I'll see if I can get this added for ya then.

**Comment by fashberg** [comment]
no, i've installed without any setup.ini file with just default option. 
"/si" didn't worked for me (installation did not happen) , but "MSEdgeRedirect.exe /wingetinstall" (ran as user SYSTEM) was fine for all users not using edge

**Comment by rcmaehl** [comment]
The Edge as the default browser issue should be fixed in 0.7.5.0.

Edge as the Default PDF viewer is still being worked on but should be fixed with PerUser Active mode in 0.8.0.0
